CHANGELOG.md

```diff
@@ -4,6 +4,50 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## 0.11.0 (2018-08-08)
+
+**High-level changes**
+* Hardening the database: blocking commands that do not yet have defined behavior, adding support for several previously unsupported table alterations, adding more test coverage.
```

Comment:
`blocking commands that do not yet have defined behavior`---> `blocking commands that do not make sense for hypertables`